packages/api-client-core/src/index.ts

```diff
@@ -13,5 +14,3 @@ export * from "./operationBuilders";
 export * from "./operationRunners";
 export * from "./support";
 export * from "./types";
-
-export type { AsyncRecord, AsyncRecordList, FindOneFunction, FindManyFunction, ActionFunction } from "./GadgetFunctions";
```

Comment:
I guess this still builds ok eh? I thought I had to do this to get compiling with `isolatedModules` on, but if CI is 🍏 I think we're good! 👍

Reply:
Hmm, yeah, that _could_ be problematic, but I haven't had any issues. Maybe `export *` is fine?